src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -1,73 +1,132 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import moment from 'moment';
+import appendQuery from 'append-query';
+
 import { focusElement } from '../../../common/utils/helpers';
 import OMBInfo from '../../../common/components/OMBInfo';
 import FormTitle from '../../../common/schemaform/components/FormTitle';
+import RequiredLoginView from '../../../common/components/RequiredLoginView';
+import { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import { toggleLoginModal } from '../../../login/actions';
 
-import SaveInProgressIntro, { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import formConfig from '../config/form';
+import { submitIntentToFile } from '../actions';
+import SaveInProgressIntro from './SaveInProgressIntro';
+import { UnauthenticatedAlert } from '../helpers';
 
 class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  hasSavedForm = () => {
+    const { saveInProgress: { user } } = this.props;
+    return user.profile && user.profile.savedForms
+      .filter(f => moment.unix(f.metadata.expires_at).isAfter())
+      .find(f => f.form === this.props.formId);
+  }
+
+  updateITFStatus = (ITFStatus) => {
+    this.setState(ITFStatus);
+  }
+
+  handleLoadPrefill = () => {
+    this.props.submitIntentToFile(
+      formConfig,
+      this.updateITFStatus);
+  }
+
+  authenticate = (e) => {
+    e.preventDefault();
+    const nextQuery = { next: e.target.getAttribute('href') };
+    const nextPath = appendQuery('/', nextQuery);
+    history.pushState({}, e.target.textContent, nextPath);
```

Comment:
As we talked about on the call, this may be overkill. What happens when we just do `toggleLoginModal()`?